### ISSUE TITLE
Logs filled with "Not Found"

### ISSUE THREAD

**Issue by ONordander** [polyaxon/polyaxon#1046]
### Describe the bug
When running an experiment, logs are spammed with just "Not Found". If I download the plxlogs from our artifactsStore (minIO) I can see that the logs come from the polyaxon-sidecar container.
### To Reproduce
```
version: 1.1
kind: component
name: simple-experiment
description: Minimum information to run this TF.Keras example
tags: [examples]
run:
  kind: job
  init:
  - git: {url: "https://github.com/polyaxon/polyaxon-quick-start"}
    container:
      env:
        - name: http_proxy
          value: "***"
        - name: https_proxy
          value: "***"
  container:
    image: polyaxon/polyaxon-quick-start
    workingDir: "{{ globals.artifacts_path }}/polyaxon-quick-start"
    command: [python3, model.py]
    env:
      - name: http_proxy
        value: "***"
      - name: https_proxy
        value: "***"
```

### Expected behavior
No "Not Found" logs.

### Environment
```
deploymentChart: platform
deploymentVersion: 1.1.3

artifactsStore:
  name: minio
  kind: s3
  schema: {"bucket": "***"}
  secret:
    name: "***"

connections:
  - name: data
    kind: volume_claim
    schema:
      mountPath: ***
      volumeClaim: ***
      readOnly: true

scheduler:
  enabled: true

streams:
  enabled: true

postgresql:
  persistence:
    enabled: true
    storageClass: nfs

redis:
  enabled: true
  master:
    persistence:
      enabled: true
      storageClass: nfs
  slave:
    persistence:
      enabled: true
      storageClass: nfs
broker: redis

rabbitmq-ha:
  enabled: false

adminViewEnabled: true
```

**Comment by mouradmourafiq** [comment]
Thanks for opening the issue, I have seen this in GCS and Azure, and a patch was made to silence the debug outputs, I think similar thing should be done for S3.

If there's bandwidth a fix should be part of v1.1.8.

**Comment by mouradmourafiq** [comment]
This is related to https://github.com/minio/minio/issues/6540, currently a workaround is implementing this feature #1050

**Comment by mouradmourafiq** [comment]
TODO: Explore using minio-py instead of boto for artifacts stores backed with a minio

**Comment by ONordander** [comment]
Seems to be fixed with v1.1.8 with logs from Main-container only, thanks.
Closing.

**Comment by mouradmourafiq** [comment]
I will keep this issue, since the there's still a lot of warning logs coming from the issue between boto3 and minio backend.
